### PR TITLE
dataset overview page

### LINKED ISSUE #235: Create overview page for a dataset
coming from https://github.com/girder/viime/pull/217#issuecomment-532708162

create an overview page of a dataset showing
 * name
 * size
 * upload date
 * ??

related to #197 which could be done here, too

---
> +1 to the overview page. Let me know if you need some sort of design added to the mockups for that or the sidebar, or if you think you have a clear enough path you can build it and I can provide feedback once it's ready.

### PR DESCRIPTION
**Pull request by sgratzl**
closes #235  and #197 

first simple version of a dataset overview. 
image

more properties can be added as soon as more about the validated data is available in the store (#248)

note
 * please run `flask db upgrade` to alter the DB scheme

### CODE REVIEW

**Comment by sgratzl** [comment]
> Please wrap actions in try/catch. Also, some feedback from others on the api structure is needed. No changes necessarily needed there.

wrapped

File: viime/views.py [hunk 1]
```diff
@@ -247,6 +247,38 @@ def set_csv_file_metadata(csv_id):
...
+    except Exception:
+        db.session.rollback()
+        raise
+
+
+@csv_bp.route('/csv/<uuid:csv_id>/description', methods=['PUT'])
+@use_kwargs({
+    'description': fields.Str(required=True)
+})
+def set_csv_file_description(csv_id, description):
```
[the remark this answers is not in the document]
Comment (jbeezley) [reply viime/views.py]: Yeah, this is what I recommended based on the other API calls.  You are right this unconventional; a `PUT` request is supposed to contain the full model.  This is really a `PATCH` request.  The model here is doesn't exactly satisfy the usual property of a restful resource.

What attributes would you add to `PUT /csv/:id`? name, description, imputation_m{n,c}ar, and meta?  Adding the transformation options would be more difficult due to how they interact with the validation.

[the remark this answers is not in the document]
Comment (sgratzl) [reply viime/views.py]: if you wanna change those things, please open an refactoring issue cause this is out of the scope of this PR and issue

[the remark this answers is not in the document]
Comment (jbeezley) [reply viime/views.py]: I agree.

**Pull request merged**